### PR TITLE
reset attempts on success

### PR DESCRIPTION
**Pull request by grooviegermanikus**

### CODE REVIEW

File: src/grpc_subscription_autoreconnect_streams.rs [hunk 1]
```diff
@@ -67,18 +66,18 @@ pub fn create_geyser_reconnecting_stream(
                         }
                     });
 
-                    (ConnectionState::Connecting(attempt, connection_task), Message::Connecting(attempt))
+                    (ConnectionState::Connecting(attempt + 1, connection_task), Message::Connecting(attempt + 1))
```
Comment (grooviegermanikus) [inline src/grpc_subscription_autoreconnect_streams.rs]: this might be wrong (the message part)

**Comment by godmodegalactus** [comment]
clippy

**Comment by godmodegalactus** [comment]
Looks better.

**Review by godmodegalactus — APPROVED** [review]

**Comment by grooviegermanikus** [comment]
tagged `v0.10.3+yellowstone.1.12+solana.1.17.15`

**Pull request merged**